src/System.Text.Json/tests/Serialization/Object.ReadTests.cs

```diff
@@ -230,5 +230,12 @@ public static void ReadConversionFails()
 
             Assert.True(exceptionThrown);
         }
+
+        [Fact]
+        public static void ReadObject_PublicIndexer()
+        {
+            var indexer = JsonSerializer.Parse<Indexer>(@"{""NonIndexerProp"":""Value""}");
```

Comment:
Nit: var => Indexer

Reply:
I thought that parse generics arg was enough clear, will fix